data/web/public/index.php

```diff
@@ -1,127 +1,273 @@
-            background-position: bottom;
-            background-repeat: no-repeat;
-            min-height: 600px;
-            text-align: center;
-        }
-        .jumbotron h1 {
-            font-size: 36px;
-            font-family: inherit;
-            font-weight: inherit;
-            text-align: inherit;
-        }
-        .navbar-brand img {
-            width: 70%;
+$host_name = gethostname();
+$host_name_parts = explode("-", $host_name);
+$ready_name = count($host_name_parts) == 4 ? $host_name_parts[1] : $host_name;
```

Comment:
Isn't that `4` considered a [magic number](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)?